central/enrichment/singleton.go

```diff
@@ -39,6 +40,7 @@ var (
 	once sync.Once
 
 	ie                    imageEnricher.ImageEnricher
+	ieV2                  imageEnricher.ImageEnricherV2
```

Comment:
nit:  lets make the var name better.  Better yet, fix them all as those are unreadable later when passed in.